app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -51,8 +52,10 @@ class BundledTilesManager private constructor(context: Context) {
         val lhm = LinkedHashMap<Uri, BundledHomeTile>(tilesJSONArray.length())
         for (i in 0 until tilesJSONArray.length()) {
             val tile = BundledHomeTile.fromJSONObject(tilesJSONArray.getJSONObject(i))
-            // TODO: Check for blacklisted sites and don't add them
-            lhm.put(tile.url.toUri()!!, tile)
+            val blacklist = loadBlacklist(context)
```

Comment:
Load this blacklist outside of the loop - this will load the blacklist for every item in the bundled tiles cache!